tensorflow_addons/metrics/hamming.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implements Hamming distance and loss."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+from tensorflow_addons.metrics.utils import MeanMetricWrapper
+
+
+def hamming_distance(y_true, y_pred):
+    """Computes hamming distance.
+
+    Hamming distance is for comparing two binary strings.
+    It is the number of bit positions in which two bits
+    are different.
+
+    :param y_true: actual target value
```

Comment:
Could you change the docstring to standardize it according to like `f_scores.py`? We need to change the other metrics to conform to this format as well.